db/structure.sql

```diff
@@ -31,15 +31,15 @@ SET default_with_oids = false;
 --
 
 CREATE TABLE public.addresses (
-    id bigint NOT NULL,
+    id integer NOT NULL,
```

Comment:
We should figure out why your system is creating these indices as `integers` and not `bigint`  -- in a separate PR.